src/content/guides/getting-started.md

```diff
@@ -143,6 +143,9 @@ __project__
     |- index.js
 ```
 
+T> The observant among you may have noticed that `index.html` is created manually, even though it is now placed in the `dist` directory. Later on in this guide, we will generate `index.html` rather than edit it manually. Once this is done, it should be safe to empty the `dist` directory and to regenerate all the files within it.
```

Comment:
I don't like the tone of this oration, would rather have: `You may have noticed...`

Reply:
Fair enough. I did wonder about that wording, but the pull request had already been merged by the time I thought about it.

Reply:
Feel free to submit a correction :)